tests/driver_bmx055/README.md

```diff
@@ -0,0 +1,14 @@
+# About
+This is a test application that is polling all three SAUL devices
+(Magnetometer, Accelerometer, Gyroscope) of the Bosch BMX055
+9-Axis Sensor.
+This test is a copy of the standard saul test. Since it utilizes all
```

Comment:
I would prefer a minimal test application that doesn't use the saul module but just the driver API. This is how things are done for other drivers and we want to be consistent.

Reply:
I looked over the example you provided, but i also talked to @haukepetersen about it and he recommended using the saul test. I am not in the position to decide, so I would prefer both of you decide this together. I would be okay writing another test.

Reply:
I agree with @aabadie to have a non-saul test application using the drivers _API_, for testing with SAUL there already is `tests/saul` and also `examples/saul` plus `examples/default` would also do. I mean, in the worst case this tests fails due to some issue with SAUL and not with the driver itself.